src/cargo/ops/cargo_generate_lockfile.rs

```diff
@@ -184,11 +229,34 @@ pub fn update_lockfile(ws: &Workspace<'_>, opts: &UpdateOptions<'_>) -> CargoRes
             }
         } else {
             for package in removed.iter() {
-                print_change("Removing", format!("{}", package), &style::ERROR)?;
+                print_change("Removing", format!("{package}"), &style::ERROR)?;
             }
             for package in added.iter() {
-                print_change("Adding", format!("{}", package), &style::NOTE)?;
+                print_change("Adding", format!("{package}{latest}"), &style::NOTE)?;
+            }
+        }
+        if !latest.is_empty() {
+            if opts.config.shell().verbosity() == Verbosity::Verbose {
+                for package in &unchanged {
+                    opts.config.shell().status_with_color(
+                        "Unchanged",
+                        format!("{package}{latest}"),
+                        &anstyle::Style::new().bold(),
+                    )?;
+                }
             }
+            unchanged_behind += unchanged.len();
+        }
+    }
+    if opts.config.shell().verbosity() == Verbosity::Verbose {
+        opts.config.shell().note(
+            "To see how you depend on a package, run `cargo tree --invert --package <dep>@<ver>`",
+        )?;
+    } else {
+        if 0 < unchanged_behind {
```

Comment:
Suggestion: I think writing `unchanged_behind > 0` would be more idiomatic (and easier to parse for me, at least)?

And even then, it's a little unclear to me what this condition is about.

Reply:
While there is yoda speak (`constant == variable`) and more general speak (`variable == constant`) for equality, I see comparison operators differently and prefer to write them in number line order as I find that easier to read.

Reply:
I am on the same side of djc regarding the order, but not really a blocker.